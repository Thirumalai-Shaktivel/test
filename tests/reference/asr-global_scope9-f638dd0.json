{
    "basename": "asr-global_scope9-f638dd0",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/global_scope9.f90",
    "infile_hash": "29e87c62bbe7f9efedd2e3cf348e80153367ecc0369b16de6f4db296",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-global_scope9-f638dd0.stdout",
    "stdout_hash": "2e21a337bac7306c33ced0d1701c554b1e8058bd85faafdf47783f4d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}