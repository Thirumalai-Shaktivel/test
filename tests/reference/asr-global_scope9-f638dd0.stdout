(TranslationUnit (SymbolTable 1 {x: (Variable 1 x Local () Default (Integer 8 []) Source Public Required)}) [(= (Var 1 x) (ImplicitCast (ConstantInteger 6 (Integer 4 [])) IntegerToInteger (Integer 8 []) ())) (Var 1 x)])
